src/ng-package.schema.json

```diff
@@ -8,6 +8,11 @@
     "$schema": {
       "type": "string"
     },
+    "deleteOutputPath": {
+      "description": "Delete output path before build.",
+      "type": "boolean",
+      "default": true
```

Comment:
Hi, looks good to me! Shouldn't we call it `deleteDestPath` since it's related to the `dest` path?

Reply:
I don’t mind renaming it. Originaly I choose that name due to Ng CLI

Reply:
Pushed the rename